### PR TITLE
Testsuite PoC - modules refactoring.

### LINKED ISSUE #31412: Testsuite Poc - new modules structure
It will create a new modules, which will be used within this issue: #31312

### PR DESCRIPTION
**Pull request by lhanusov**
Closes #31412

### CODE REVIEW

**Comment by stianst** [comment]
Would be nice to get https://github.com/keycloak/keycloak/pull/31406 merged first if that's okay

**Review by vaceksimon — APPROVED** [review]
LGTM

**Comment by jonkoops** [comment]
@lhanusov can you fix the merge conflicts on this one?

File: test-poc/framework-tests/pom.xml
Comment (stianst) [inline test-poc/framework-tests/pom.xml:1]: We don't need this module, at least not for now. I've added framework tests as unit tests to framework instead for now.

Comment (lhanusov) [reply test-poc/framework-tests/pom.xml:1]: removed :+1:

**Review by stianst — APPROVED** [review]

**Pull request merged**